.gitlab-ci.yml

```diff
@@ -225,7 +225,7 @@ terraform:
   variables:
     <<:                            *default-vars
     # https://releases.hashicorp.com/terraform/
-    TERRAFORM_VERSION:             "1.0.2"
+    TERRAFORM_VERSION:             "1.0.3"
     # https://releases.hashicorp.com/vault/
     VAULT_VERSION:                 "1.6.3"
```

Comment:
Vault is [already](https://www.vaultproject.io/downloads) 1.8, maybe bump it as well?

Reply:
Good point, I have ugpraded Vault as well.